exampleSite/config.toml

```diff
@@ -70,6 +70,10 @@ paginate = 10
     defaultKeywords = ["devows", "hugo", "go"]
     defaultDescription = "Site template made by devcows using hugo"
 
+    # Twitter
+    twitter = "GoHugoIO"
+    twitterImage = "img/twitter-default.png"
```

Comment:
People won't always use local urls, have you made sure this works with external urls too?

Reply:
Good suggestion, `absURL` should work well.

Reply:
I forget, this should work with either/or now then, can you quickly test it with a local and again with a remote?